webapp/app.py

```diff
@@ -232,37 +232,45 @@ def careers_index():
     all_departments = (_group_by_department(greenhouse.get_vacancies()),)
 
     dept_list = [
-        "engineering",
-        "support-engineering",
-        "marketing",
-        "web-and-design",
-        "project-management",
-        "commercial-operations",
-        "product",
-        "sales",
-        "finance",
-        "people",
-        "administration",
-        "legal",
+        {"slug": "engineering", "icon": "84886ac6-Engineering.svg"},
```

Comment:
Should we not do this in the department object instead? Some of these slugs are already redefined there https://github.com/canonical/canonical.com/blob/9a61c0db964d049e58f8292fa12efa7fef961d35/webapp/greenhouse.py#L81

Reply:
The department object only knows about information in Greenhouse. The icons data is not stored in Greenhouse and so the `dept_list` is the only place we filter out departments to the set we want to render on the site.